### PR TITLE
fix(formatting): Zeckendorf challenge | Rosetta code

### PR DESCRIPTION
**Pull request by thecodingaviator**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/zeckendorf-number-representation.english.md [hunk 1]
```diff
@@ -6,36 +6,13 @@ challengeType: 5
...
-restriction that ''no two consecutive Fibonacci numbers can be used''
-which leads to the former unique solution.
-</p>
-<p>
- Task:
- Write a function that generates and returns an array of first N Zeckendorf numbers in order.
-</p>
+Just as numbers can be represented in a positional notation as sums of multiples of the powers of ten (decimal) or two (binary); all the positive integers can be represented as the sum of one or zero times the distinct members of the Fibonacci series. Recall that the first six distinct Fibonacci numbers are:  <code>1, 2, 3, 5, 8, 13</code>.
+The decimal number eleven can be written as <code>0*13 + 1*8 + 0*5 + 1*3 + 0*2 + 0*1</code> or <code>010100</code> in positional notation where the columns represent multiplication by a particular member of the sequence. Leading zeroes are dropped so that 11 decimal becomes <code>10100</code>. 10100 is not the only way to make 11 from the Fibonacci numbers however <code>0*13 + 1*8 + 0*5 + 0*3 + 1*2 + 1*1</code> or 010011 would also represent decimal 11. For a true Zeckendorf number there is the added restriction that 'no two consecutive Fibonacci numbers can be used' which leads to the former unique solution.
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/zeckendorf-number-representation.english.md]: Could we italicize some of the text here?:

`<i>no two consecutive Fibonacci numbers can be used</i>`

File: curriculum/challenges/english/08-coding-interview-prep/rosetta-code/zeckendorf-number-representation.english.md [hunk 2]
```diff
@@ -6,36 +6,13 @@ challengeType: 5
...
 </section>
 
 ## Instructions
 <section id='instructions'>
-
+Write a function that generates and returns an array of first N Zeckendorf numbers in order.
```
Comment (scissorsneedfoodtoo) [inline curriculum/challenges/english/08-coding-interview-prep/rosetta-code/zeckendorf-number-representation.english.md]: And I thought we could change this slightly:

`Write a function that generates and returns an array of the first <code>n</code> Zeckendorf numbers in order.`

**Comment by thecodingaviator** [comment]
@scissorsneedfoodtoo Changes made

**Comment by scissorsneedfoodtoo** [comment]
@thecodingaviator, everything LGTM! Will ask in the Gitter chat for another review.

**Review by ezioda004 — APPROVED** [review]

**Comment by ezioda004** [comment]
Thanks @thecodingaviator for fixing this!

**Pull request merged**